torch/nn/modules/module.py

```diff
@@ -1227,6 +1258,11 @@ def _save_to_state_dict(self, destination, prefix, keep_vars):
         for name, buf in self._buffers.items():
             if buf is not None and name not in self._non_persistent_buffers_set:
                 destination[prefix + name] = buf if keep_vars else buf.detach()
+        extra_state_key = prefix + _EXTRA_STATE_KEY_SUFFIX
+        if getattr(self.__class__, "get_extra_state", Module.get_extra_state) is not Module.get_extra_state:
+            destination[extra_state_key] = OrderedDict()
+            for name, obj in self.get_extra_state().items():
+                destination[extra_state_key][name] = obj
```

Comment:
Why do you write these items by hand instead of just doing `destination[extra_state_key] = self.get_extra_state()` ?

Reply:
Fixed